### ISSUE TITLE
Where is the mask consistency loss?

### ISSUE THREAD

**Issue by DailyVy** [hustvl/MaskAdapter#4]
Hello, I read your paper that really impressed me. I was looking for the part of mask consistency loss but I figured there is only cross entropy loss in  [           losses = self.cross_entropy_loss(mask_cls_results, labels)](https://github.com/hustvl/MaskAdapter/blob/cf987a737eec62ff32183537c6ffcef0164c01fa/mask_adapter/mask_adapter.py#L374)
Could you tell me where the mask consistency loss is?

**Comment by DailyVy** [comment]
Thank you! When calculating cross-entropy loss, is the predicted mask ever used as a label, or is only the ground truth label used?

**Comment by owl-10** [comment]
When calculating the cross-entropy loss, ground-truth uses its own ground-truth labels, while predicted masks use the labels of the ground-truth masks matched by the IoU-based matcher.
